metals/src/main/scala/scala/meta/internal/parsing/FoldingRangeExtractor.scala

```diff
@@ -106,6 +108,22 @@ final class FoldingRangeExtractor(
     foldingRange
   }
 
+  /**
+   * Due to logic thats forcibly shifts position in `Foldable`
+   * `end` might point at `\r>>range.end<<\n`
+   * that causes problems on Windows (at least in tests)
```

Comment:
I wonder why this is popping up now?

Reply:
This might have been an issue before, but only popped up recently due to my changes.

Reply:
Too late, I will take a look at it.

Reply:
Opps, sorry)

Yep, I thought that it would be better to check how it works in the editor but I was too lazy to install Windows VM.
However, to be sure we have to test it on several editor.

Reply:
I have a Windows machine set up and will test it out. Shouldn't take long to look at it. Thanks for fixing this btw!